pkg/environment/environment.go

```diff
@@ -188,4 +244,9 @@ func LogEnvVariables(log *zerolog.Logger) {
 	for _, envVar := range envVarArray {
 		logEnvVariable(log, envVar)
 	}
+
```

Comment:
GetDataPathMaxSize ?

Reply:
the error of GetDataPathMaxSize if such exists is printed in FindPaths().

Reply:
Yes.